include/stumpless/version.h

```diff
@@ -44,6 +44,21 @@ struct stumpless_version {
 struct stumpless_version *
 stumpless_get_version( void );
 
+/**
+ * Compares the version of two given stumpless_version struct.
```

Comment:
Make sure that the function returns a positive _or_ negative value depending on which of the two arguments is larger: negative if the first is smaller, positive if the first is larger. This is the same convention used by the standard library's `strcmp` function (some good documentation on it [here](http://www.cplusplus.com/reference/cstring/strcmp/)), and so should be familiar to users.

This behavior is important because it allows the caller to know which of the two versions is greater, so that they can make decisions such as "it's fine if these aren't equal as long as the actual version is higher than what I expected".